### DISCUSSION TITLE
Daily and weekly view with time-grid

### DISCUSSION THREAD

**Discussion by acsala** [kiwanami/emacs-calfw#170] [Ideas]
I am looking for an option to display daily and weekly views with time-grids, like in org agenda time-grid style (e.g.; see figure below).

I wonder if someone already implemented this functionality?

![image](https://user-images.githubusercontent.com/4764550/159428274-4b949676-aed5-44d2-bdae-09469ee268f9.png)

**Comment by haji-ali** [comment]
This would go against the block like structure of `calfw`. Search for `calfw-block` (e.g., my fork or the original) for a similar output.
